elrond-config.md

```diff
@@ -414,18 +416,36 @@ TODO: Implement [reserved keys and read-only runtimes](https://github.com/Elrond
 ```k
     syntax InternalCmd ::= "#endWasm"
  // ---------------------------------
-    rule <commands> #endWasm => popCallState ~> dropWorldState ... </commands>
+    rule <commands> #endWasm => #asyncExecute ~> #setVMOutput ~> popCallState ~> dropWorldState ... </commands>
          <instrs> .K </instrs>
-         <out> OUT </out>
-         <logs> LOGS </logs>
-         <vmOutput> _ => VMOutput( OK , .Bytes , OUT , LOGS) </vmOutput>
       [priority(60)]
 
     syntax InternalCmd ::= "#waitWasm"  [klabel(#waitWasm), symbol]
  // ----------------------------------
     rule <commands> #waitWasm => .K ... </commands>
          <instrs> .K </instrs>
       [priority(60)]
+
+    syntax InternalInstr ::= "#waitCommands"
+ // ---------------------------------------
+    rule [waitCommands]:
+        <instrs> #waitCommands => .K ... </instrs>
+        <commands> #endWasm ... </commands>
+      [priority(200)]
+
+    rule <commands> #transferSuccess => .K ... </commands>
+         <instrs> #waitCommands ... </instrs>
+    
+
+    syntax InternalCmd ::= "#setVMOutput"
+ // -------------------------------------
+    rule <commands> #setVMOutput => .K ... </commands>
+         <out> OUT </out>
+         <logs> LOGS </logs>
+         <outputAccounts> OUT_ACCS</outputAccounts>
+         <vmOutput> _ => VMOutput( OK , .Bytes , OUT , LOGS , OUT_ACCS) </vmOutput>
```

Comment:
Either remove the space after '(' or add one before ')'.

This applies everywhere, I guess.